slicedimage/_collection.py

```diff
@@ -0,0 +1,36 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from ._tileset import TileSet
+
+
+class Collection(object):
+    def __init__(self, extras=None):
+        self.extras = extras
+        self._partitions = dict()
```

Comment:
self._tilesets? in general, we shouldn't have the word partition anymore in the codebase right?

Reply:
arg. i get it. this could either be a collection or a tileset. i'd just leave a comment about that here and move on until we can come up with a better word than partition